src/actions.js

```diff
@@ -25,14 +25,22 @@ module.exports = {
       console.log('Please use a correct date in the form of YYYY-MM-DD')
       return
     }
-    let newWorkLogPromise = jira.addNewWorklog(issueId, time, message, options.date)
-    newWorkLogPromise.then((ids) => {
-      console.log(ids.issueId + ' ' + ids.worklogId)
-      process.exit(0)
-    }).catch((err) => {
-      console.log('Work was not logged successfully')
-      console.error(err)
-      process.exit(1)
+
+    let issuesPromise = jira.getUsersIssues()
+    issuesPromise.then((issues) => {
+      const issueIds = issues.map((issue) => { return issue.key })
+      if (!(issueId in issueIds)) {
```

Comment:
This line isn't doing what you think. The `in` operator in JavaScript checks to see if an object has a property. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in

Reply:
In other words, you'll see the warning for issue instead regardless of whether or not it's yours.